packages/experimental/Button/src/Button.tsx

```diff
@@ -3,12 +3,23 @@ import * as React from 'react';
 import { View } from 'react-native';
 import { buttonName, ButtonType, ButtonProps } from './Button.types';
 import { Text } from '@fluentui-react-native/experimental-text';
-import { stylingSettings } from './Button.styling';
+import { stylingSettings, getDefaultSize } from './Button.styling';
 import { compose, mergeProps, withSlots, UseSlots } from '@fluentui-react-native/framework';
 import { useButton } from './useButton';
 import { Icon } from '@fluentui-react-native/icon';
 import { createIconProps } from '@fluentui-react-native/interactive-hooks';
 
+export const buttonLookup = (layer: string, state: ButtonState, userProps: ButtonProps): boolean => {
```

Comment:
This was getting large and we would want to share it across the various button types, so I refactored it out into its own function

Reply:
Added comment to the function.